ci-operator/step-registry/cucushift/upgrade/check/cucushift-upgrade-check-commands.sh

```diff
@@ -34,10 +34,12 @@ if [ -z "${UPGRADE_SKIP_TAGS}" ] ; then
 else
     export UPGRADE_SKIP_TAGS="${UPGRADE_SKIP_TAGS} and not @customer and not @security"
 fi
+UPGRADE_SKIP_TAGS_CLOUD="${UPGRADE_SKIP_TAGS/and not @destructive/}"
 
 cd verification-tests
 set -x
 cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS}" -p junit || true
+cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS_CLOUD} and @cloud and @destructive" -p junit || true
```

Comment:
@sunzhaohua2 Can we restrict it to only the cloud scenario by `-n SCENARIO_NAME`? `@cloud and @destructive` gives us test to autoscaler and health check.

Reply:
```
UPGRADE_CHECK_RUN_TAGS='@upgrade-check and @4.12 and @aws-ipi and not @fips'
UPGRADE_SKIP_TAGS_CLOUD='not @customer and not @flaky and not @inactive and not @prod-only and not @qeci and not @security and not @stage-only'
```

`cucumber --tags "${UPGRADE_CHECK_RUN_TAGS} and ${UPGRADE_SKIP_TAGS_CLOUD} and @destructive"` return 7 scenarios.
If we add `and @cloud`, the scenario number is controllable.

Reply:
@sunzhaohua2   Could you help add tag `@cloud` to the test cases you want to execute for the post upgrade in verification-tests and cucushift repository ?  Thanks.

Reply:
https://github.com/openshift/verification-tests/pull/3259
```
$ bundle exec cucumber --tags '@upgrade-check and @4.12 and @aws-ipi and not @fips and not @customer and not @flaky and not @inactive and not @prod-only and not @qeci and not @security and not @stage-only and @cloud' --dry-run
1 scenario (1 skipped)
```